componentLibraries/extensionLibrary/extensionLibrary.pro

```diff
@@ -15,6 +15,8 @@ DESTDIR = $${PWD}/
 # *= Means append unique
 INCLUDEPATH *= $${PWD}/../../HopsanCore/include/
 
+INCLUDEPATH *= $${PWD}/../../Dependencies/sundials-5.3.0/include/
```

Comment:
If we hide sundiles inside hopsancore then the component library do not need to know about the library,
only the utility that we expose that wraps it

Reply:
I think this one can also be removed